lib/evm_database.rb

```diff
@@ -16,6 +16,12 @@ class EvmDatabase
     MiqReport
     RssFeed
     MiqWidget
+    MiqAction
+    MiqEventDefinition
+    Condition
+    MiqPolicy
+    MiqPolicyContent
+    MiqPolicySet
```

Comment:
This also creates order, although I understand it was not meant for it.

Should I add a ORDERED_CLASSES after PRIMORDIAL_CLASSES and use it [here](https://github.com/ManageIQ/manageiq/blob/master/lib/evm_database.rb#L50)
in a different pr?